docker/docker.go

```diff
@@ -251,6 +257,7 @@ func (i *Image) Pull() error {
 		}
```

Comment:
@FrimIdan I see that for some errors you get the dump and for others don't. How do you decide?

Reply:
I tried to dump the response. Here the request was unsuccessful so the URL is printed inside the pullReq() func.